hedera-mirror-importer/src/main/resources/db/scripts/bootstrap.sh

```diff
@@ -11,7 +17,14 @@ exec 1>/dev/null 2>>bootstrap.log
 exec 3>&2
 exec 2>/dev/null
 
-PID_FILE="bootstrap.pid"
+SCRIPT_TEMP_DIR="./temp"
+
+if [[ -d "$SCRIPT_TEMP_DIR" ]]; then
+  rm -rf "$SCRIPT_TEMP_DIR"
+fi
```

Comment:
nit: rm -rf works if dir doesn't exist so could simplify

```suggestion
rm -rf "$SCRIPT_TEMP_DIR"
```